tests/TestRunner/tmp_cluster.py

```diff
@@ -62,6 +64,7 @@ def __enter__(self):
 
     def __exit__(self, xc_type, exc_value, traceback):
         super().__exit__(xc_type, exc_value, traceback)
+        self.check_trace()
```

Comment:
Will be done

Reply:
Btw it needs to be a callback because there are usage cases when we need to narrow down the trace checks to a specific time period: e.g. in the event sequence `[begin] - [call API wrong] - [get client error] - [end]`, we want to check that error trace was written with `Time` value between `[begin]` and `[end]`, otherwise the test loses accuracy.

Reply:
Also, `fdbmonitor` doesn't use XML, so it helps to keep the Sev40 with your `grep`-based implementation to capture any unexpected server crashes.